src/molecule/api.py

```diff
@@ -36,6 +36,12 @@ def append(self, item) -> None:
         super(UserListMap, self).append(item)
 
 
+class MoleculeRuntimeWarning(RuntimeWarning):
+    """A runtime warning used by Molecule and its plugins."""
+
+    ...
```

Comment:
This is unnecessary because the docstring is already present and it's the first statement in the class body.
```suggestion
```